packages/snap-preact-demo/tests/cypress/integration/autocomplete/autocomplete.spec.js

```diff
@@ -163,26 +151,23 @@ describe('Autocomplete', () => {
 		});
 
 		it('can hover over facet', function () {
-			if (!config?.selectors?.input && !config?.selectors?.autocomplete?.facet) this.skip();
-
 			cy.get(config.selectors.website.input).first().should('exist').clear({ force: true }).type(config.startingQuery, { force: true });
 			cy.wait('@autocomplete').should('exist');
 
 			cy.snapController('autocomplete').then(({ store }) => {
-				if (store.facets.length == 0) this.skip(); //skip if this term has no facets
-				cy.get('body').then((body) => {
-					if (!body.find(`${config.selectors.autocomplete.facet} a`).length) {
-						this.skip(); // skip if no facets in DOM
-					}
-				});
+				if (store.facets.length == 0) {
+					//if this term has no facets lets try another
+					cy.get(config.selectors.website.input).first().clear({ force: true }).type(config.query, { force: true });
+					cy.wait('@autocomplete').should('exist');
+				}
 
 				cy.get(`${config.selectors.autocomplete.facet} a`).then((facetOptions) => {
 					const firstOption = facetOptions[0];
 					const optionURL = firstOption.href;
-
+					cy.intercept(/.*searchspring.io\/api\/search\/autocomplete.json?.*&filter.*/).as('autocompletefiltered');
```

Comment:
Instead of this intercept, I think we could resolve the issues here by adjusting the `snapController` Cypress command.